lib/init.js

```diff
@@ -78,13 +80,15 @@ export default {
             });
           } else {
             // Regular lint warning
-            messages.push({
-              type: 'Warning',
-              severity: 'warning',
-              filePath,
-              text: match[2],
-              range: helpers.rangeFromLineNumber(textEditor, line),
-            });
+            if (!globalRegex.test(match[2]) && !globalsOmitted) {
```

Comment:
This should be `||`.

As it currently stands, if `match[2]` contains `accessing global` it will _always_ be ignored. If it _doesn't_ contain `accessing global` then the `globalsOmitted` option being false lets it show up, but if somebody enabled it then all messages will be hidden.